demisto_sdk/commands/download/downloader.py

```diff
@@ -136,7 +137,13 @@ def fetch_custom_content(self) -> bool:
                 file_name: str = self.update_file_prefix(member.name.strip('/'))
                 file_path: str = os.path.join(self.custom_content_temp_dir, file_name)
                 with open(file_path, 'w') as file:
-                    file.write(tar.extractfile(member).read().decode('utf-8'))
+                    extracted_file = tar.extractfile(member)
+                    # File might empty
+                    if extracted_file:
+                        string_to_write = extracted_file.read().decode('utf-8')
+                        file.write(string_to_write)
+                    else:
+                        raise FileNotFoundError(f'Could not extract files from tar file: {file_path}')
```

Comment:
i think we need to print error here and return false
@guyfreund please review

Reply:
There's no need for this error handling since the file doesn't exist, we are creating it here.

Reply:
it returns false in the catch.
MYPY is yelling on this